app/views/claims/_ineligibility_reason_all_schools_ineligible.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1 class="govuk-heading-xl">
```

Comment:
If we rename this to something more generic it could be more easily reused in other circumstances where a generic ineligible reason needs to be given.

Reply:
Cool - I've just renamed it to "_ineligible" and then render it in the schools ineligibility partial if the `tried_all_schools` param is set, which also solves some of the issues you highlighted below.